### PR TITLE
add proposal of install openyurt components using dashboard

### PR DESCRIPTION
**Pull request by 401lrx**
What type of PR is this?
/kind documentation

What this PR does / why we need it:
Add proposal of install openyurt components using dashboard

Which issue(s) this PR fixes:
Fixes #

Special notes for your reviewer:
@luc99hen 

Does this PR introduce a user-facing change?

### CODE REVIEW

File: docs/proposals/20230820-install-openyurt-components-using-dashboard.md [hunk 1]
```diff
@@ -0,0 +1,114 @@
...
+<div align="center">
+  <img src = "../img/dashboard-install-modal.png" width="800">
+</div>
+<div align="center">
+  <img src = "../img/dashboard-config-modal.png" width="800">
+</div>
+
+##### The first login screen for users
+
+When visit the page for the first time after installing the dashboard, a guide page will be displayed. Check the components and click Next to complete the installation of the component.
```
Comment (luc99hen) [inline docs/proposals/20230820-install-openyurt-components-using-dashboard.md:114]: Could you provide an UI preview here for this new login page?

**Review by luc99hen — APPROVED** [review]

**Comment by luc99hen** [comment]
@rambohe-ch PTAL

**Review by rambohe-ch — APPROVED** [review]

**Pull request merged**